xrpl_types/src/transaction.rs

```diff
@@ -27,6 +27,12 @@ pub enum TransactionType {
     DepositPreauth = 19,
     TrustSet = 20,
     AccountDelete = 21,
+    SetHook = 22,
```

Comment:
Where did you find these values?

Reply:
I found them here: https://github.com/XRPLF/xrpl.js/blob/main/packages/ripple-binary-codec/src/enums/definitions.json#L2303 

and confirmed here: https://github.com/XRPLF/rippled/blob/e32bc674aa2a035ea0f05fe43d2f301b203f1827/src/ripple/protocol/TxFormats.h#L56
will ad these sources as an inline comment in a follow up PR

Reply:
Thanks.